lab_1/main.py

```diff
@@ -105,7 +165,11 @@ def write_to_file(path_to_file: str, content: list):
     """
     Writes the result in a file
     """
-    pass
+    with open(path_to_file, 'w', encoding='utf-8') as file:
```

Comment:
Usually, it is a bad practice to use Python built-ins like str, list, file as variable names.